Graphics/GraphicsEngineVulkan/include/VulkanUtilities/VulkanHeaders.h

```diff
@@ -34,6 +34,10 @@
 
 #if DILIGENT_USE_VOLK
 #    include "volk/volk.h"
+#else
+// Don't use extensions when statically linked with Vulkan
```

Comment:
Why?

Reply:
Extensions doesn't exists in vulkan-1.lib, they must be loaded by vkGetDeviceProcAddr. If you use functions from extension without VOLK it causes to linker errors with unresolved symbols